requirements.txt

```diff
@@ -4,7 +4,7 @@ messytables==0.15.2
 progressbar==2.3
 #SQLAlchemy>=0.6.6 #in ckancore
 #requests==2.11.1 #in ckancore
-#six==1.11.0 #in ckancore
+six>=1.0.0 #in ckancore
 
--e git+https://github.com/qld-gov-au/ckanext-archiver.git@2.1.0-qgov.1#egg=ckanext-archiver
--e git+https://github.com/qld-gov-au/ckanext-report.git@0.1#egg=ckanext-report
+ckanext-archiver
```

Comment:
can you add it to dev requirements or does upstream have our changes now?

Reply:
The README for this extension says to install archiver and report first (which makes sense, as they're installed from source not from pypi). So the requirements file shouldn't install them, just verify.

Reply:
do add, for cicd integration testing to work. unless there is some other script already added it.

Reply:
The Travis config already installs them manually.